chain/network/src/peer_manager.rs

```diff
@@ -202,6 +227,92 @@ impl PeerManagerActor {
         })
     }
 
+    fn broadcast_accounts(
+        &mut self,
+        ctx: &mut Context<PeerManagerActor>,
+        accounts: Vec<AnnounceAccount>,
+    ) {
+        if !accounts.is_empty() {
+            debug!(target: "network", "{:?} Received new accounts: {:?}", self.config.account_id, accounts);
+        }
+        for account in accounts.iter() {
+            self.routing_table.add_account(account.clone());
+        }
+
+        let new_data = SyncData { edges: Default::default(), accounts };
+
+        if !new_data.is_empty() {
+            self.broadcast_message(
+                ctx,
+                SendMessage { message: PeerMessage::RoutingTableSync(new_data) },
+            )
+        };
+    }
+
+    fn broadcast_edges(&mut self, ctx: &mut Context<PeerManagerActor>) {
+        let me = self.peer_id.clone();
+
+        let start = Instant::now();
+        let mut new_edges = Vec::new();
+        while let Some(edge) = self.routing_table_exchange_helper.edges_to_add_receiver.pop() {
+            if let Some(cur_edge) =
+                self.routing_table.get_edge(edge.peer0.clone(), edge.peer1.clone())
+            {
+                if cur_edge.nonce >= edge.nonce {
+                    // We have newer update. Drop this.
+                    continue;
+                }
+            }
+            // Add new edge update to the routing table.
+            self.process_edges(ctx, vec![edge.clone()]);
+            if let Some(other) = edge.other(&me) {
+                // We belong to this edge.
+                if self.active_peers.contains_key(&other) {
+                    // This is an active connection.
+                    match edge.edge_type() {
+                        EdgeType::Added => {}
+                        EdgeType::Removed => {
+                            // Try to update the nonce, and in case it fails removes the peer.
+                            self.try_update_nonce(ctx, edge.clone(), other);
+                            continue;
+                        }
+                    }
+                } else {
+                    match edge.edge_type() {
+                        EdgeType::Added => {
+                            self.wait_peer_or_remove(ctx, edge.clone());
+                            continue;
+                        }
+                        EdgeType::Removed => {}
+                    }
+                }
+            }
+            new_edges.push(edge);
+            if start.elapsed() >= Duration::from_millis(50) {
```

Comment:
let's make this an associated constant of `PeerManagerActor`. Also might be better to use 100ms

Reply:
@bowenwang1996 We can configure it as we want. I configured it to do 50ms of work with 50ms delay.

If you prefer I can set it to do 100ms of work, with 50ms rest in between. 
Though I think it's better to do less work more often, for example 20 ms of work with 1ms rest.

Reply:
Okay that's fine